test/Dialect/FIRRTL/errors.mlir

```diff
@@ -155,6 +155,15 @@ firrtl.module @Foo() {
 
 // -----
 
+firrtl.circuit "Foo" {
+firrtl.module @Foo() {
+  // expected-error @+1 {{constant too large for result type}}
```

Comment:
Nit: the error message is a little unfortunate because here it's actually too small to fit

Reply:
Haha yeah good point. Magnitude is too large. "constant is too magnific ..."?

Reply:
It's too large in magnitude? :upside_down_face: 
But yeah, out of range/cannot fit anyway.